homepage/de.js

```diff
@@ -0,0 +1,74 @@
+  "Personal pricing": "Preise für Einzelpersonen",
+  "Free": "Kostenfrei",
+  "Forever": "Immer",
+  "<b>Unlimited</b> public rooms": "<b>Unbegrentzt</b> öffentliche Chat-Räume",
+  "<b>1</b> private room": "<b>1</b> privater Chat-Raum",
+  "<b>2 weeks’</b> chat history": "<b>2 Wochen</b> Chat Historie",
+  "<b>Get started</b> for free": "<b>Jetzt kostenfrei starten</b>",
+  "Individual": "Einzelperson",
+  "monthly": "monatlich",
+  "<b>Unlimited</b> private rooms": "<b>Unbegrentzt</b> private Chat-Räume",
+  "<b>Unlimited</b> chat history": "<b>Unbegrentzte</b> Chat Historie",
+  "<b>Purchase</b> now": "Jetzt <b>erwerben</b>",
+  "Organisation pricing": "Preise für Unternehmen",
+  "Bronze": "Bronze",
+  "per user/month": "pro Anwender/Monat",
+  "<b>6 months’</b> chat history": "<b>6 Monate</b> Chat Historie",
+  "Silver": "Silber",
+  "Our free plan includes one private room per organisation you belong to on GitHub": "Das kostenfreie Angebot erlaubt einen geschützen Chat-Raum pro Unternehmen dem Du auf GitHub zgeordnet bist",
+  "Our Individual plan allows you to create private rooms using your personal account as the owner": "Das Angebot für Einzelpersonen erlaubt es, geschützte Chat-Räume im Namen Deines persönlichen Benutzerkontos zu erstellen",
+  "Our organisation plans allow you to create private rooms using your GitHub organisation as the owner": "Das Angebot für Unternehmen erlaubt es, geschützte Chat-Räume im Namen Deines GitHub Unternehmens zu erstellen",
+  "Prices may be subject to VAT. By signing up you agree to our <a %s>Terms and Conditions</a>.": "Angegebene Preise unterliegen ggf. noch der gesetztlichen Mehrwertsteuer. Durch Deine Anmeldung stimmst Du unseren <a %s>allgemeinen Geschäftsbedingungen</a> zu.",
+  "Get Gitter now!": "Nutze Gitter jetzt!",
+  "Gitter is available in all modern browsers as well as apps for desktops and mobile phones.": "Gitter ist verfügbar für alle modernen Browsern und den zusätzlichen Apps für Desktop und Smartphone.",
+  "We currently have apps for OSX, iOS, Android, Windows and Linux. We also have a sweet little <a %s>IRC bridge</a>": "Zur Zeit verfügen wir über Apps für OSX, iOS, Android, Windows und Linux. Zusätzlich gibt es auch eine kleine schlanke <a %s>IRC Bridge</a>",
+  "Mac": "Mac",
+  "Windows": "Windows",
+  "Linux 32": "Linux 32",
+  "Linux 64": "Linux 64",
+  "iPhone": "iPhone",
+  "Android": "Android",
+  "By developers, for developers": "Von Entwicklern, für Entwickler",
+  "Gitter is built in London by a team of developers just like you. Hell even the product guy writes code, although we generally end up rewriting those bits.": "Gitter wurde von einer Gruppe von Entwicklern wie Dir in London gebaut. Selbst der Produkt-Mensch schreibt Code, obwohl das doch immer damit endet, dass wir diese Bits nochmal umschreiben.",
```

Comment:
[#13](https://github.com/gitterHQ/gitter-translations/pull/13/files#diff-02133d9a97f86dc218a62446d3e99843R59)

"Gitter wird von einer Gruppe von Entwicklern wie Dir in London gebaut. Selbst der Produktmanager schreibt Code, obwohl das immer damit endet, dass wir diese Bereiche nochmal umschreiben."